### ISSUE TITLE
Indsæt scrip til tracking fra Active Campaign

### ISSUE THREAD

**Issue by mtolbol** [mtolbol/sharetribe#35]
Vi ønsker at tilføje tracking af vores kontakter til vores CRM system Active campaigns, på vores sitet. Derfor ønskes følgende script indsat, i footeren på vores site:

<script type="text/javascript">
    (function(e,t,o,n,p,r,i){e.visitorGlobalObjectAlias=n;e[e.visitorGlobalObjectAlias]=e[e.visitorGlobalObjectAlias]||function(){(e[e.visitorGlobalObjectAlias].q=e[e.visitorGlobalObjectAlias].q||[]).push(arguments)};e[e.visitorGlobalObjectAlias].l=(new Date).getTime();r=t.createElement("script");r.src=o;r.async=true;i=t.getElementsByTagName("script")[0];i.parentNode.insertBefore(r,i)})(window,document,"https://diffuser-cdn.app-us1.com/diffuser/diffuser.js","vgo");
    vgo('setAccount', '252316859');
    vgo('setTrackByDefault', true);

    vgo('process');
</script>

![66786721_441349726706562_1323801490057330688_n](https://user-images.githubusercontent.com/44906533/60890205-2354e280-a25b-11e9-9e51-e9df53f530ed.png)

**Comment by koppen** [comment]
Hvis jeg ikke tager meget fejl er det muligt at gøre fra admin-interfacet et eller andet sted; mener bestemt jeg så et tekst/kode-felt, hvor man kunne indsætte den slags scripts i, som så ville blive indsat på alle sider.

**Comment by mtolbol** [comment]
Det tror jeg muligvis du har ret i. Tænker du på nedenstående "Custom script" felt? 

Bør denne kunne bruges til formålet?

![Custom script - Sharetribe](https://user-images.githubusercontent.com/44906533/61106724-457a7a80-a47e-11e9-812b-c334ee68d004.PNG)

**Comment by koppen** [comment]
Ja, lige nøjagtigt. Copy/paste scriptet der så burde det fungere. Og er der
flere af den slags, bør de kunne indsættes efter hinanden i samme felt.

Det sagt, så burde vi nok overveje at sætte Google Tag Manager op, så vil
vi kunne indsætte scripts af den her slags uden at skulle deploye nye
versioner af sitet.

- Jakob

Den 12. juli 2019 kl. 08.23.08, Morten Tølbøl Sørensen (
notifications@github.com) skrev:
